zero-trust-whitepaper/v1/cloud-native-zero-trust-whitepaper.md

```diff
@@ -0,0 +1,591 @@
+* Essential Eight Maturity Model (beyondtrust.com): [https://www.beyondtrust.com/blog/entry/essential-eight-maturity-model-updates](https://www.beyondtrust.com/blog/entry/essential-eight-maturity-model-updates)
+* Cybersecurity Policies (Europe): [https://digital-strategy.ec.europa.eu/en/policies/cybersecurity-policies](https://digital-strategy.ec.europa.eu/en/policies/cybersecurity-policies)
+* Success Story: Israel National Cyber Directorate Version 2.0 | NIST: [https://www.nist.gov/cyberframework/success-stories/israel-national-cyber-directorate-version-20](https://www.nist.gov/cyberframework/success-stories/israel-national-cyber-directorate-version-20)
+* Government Zero Trust Architecture (GovZTA) | Singapore Government Developer Portal (tech.gov.sg): [https://www.developer.tech.gov.sg/guidelines/standards-and-best-practices/government-zero-trust-architecture](https://www.developer.tech.gov.sg/guidelines/standards-and-best-practices/government-zero-trust-architecture)
+* 2022-2023 Best Undergraduate Cybersecurity Programs - US News Rankings: [https://www.usnews.com/best-colleges/rankings/computer-science/cybersecurity](https://www.usnews.com/best-colleges/rankings/computer-science/cybersecurity)
+* Referenced Projects:
+  * CNCF Knative’s Security-Guard: [https://knative.dev/docs/serving/app-security/security-guard-about/](https://knative.dev/docs/serving/app-security/security-guard-about/)
+  * Falco: [https://falco.org/](https://falco.org/)
+  * Cilium: [https://cilium.io/](https://cilium.io/)
+  * Pixie: [https://docs.px.dev/](https://docs.px.dev/)
+  * KubeArmor: [https://kubearmor.io/](https://kubearmor.io/)
+  * Curiefense: [https://www.curiefense.io/](https://www.curiefense.io/)
+  * Istio: [https://istio.io/](https://istio.io/)
+  * Linkerd: [https://linkerd.io/](https://linkerd.io/)
+  * Dapr: [https://dapr.io/](https://dapr.io/)
+  * Knative: [https://knative.dev/](https://knative.dev/)
+  * Dex: [https://dexidp.io/](https://dexidp.io/)
+  * Keycloak: [https://www.keycloak.org/](https://www.keycloak.org/)
+  * SPIFFE and SPIRE: [https://spiffe.io/](https://spiffe.io/)
+  * JSON Web Tokens (JWT): [https://jwt.io](https://jwt.io)
+  * Cert-manager: [https://cert-manager.io/](https://cert-manager.io/)
+  * Kubernetes Network Policies: [https://kubernetes.io/docs/concepts/services-networking/network-policies/](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+
+### History of Zero Trust:
+
+* [https://infraon.io/blog/history-of-zero-trust-security/](https://infraon.io/blog/history-of-zero-trust-security/) <!-- cspell:enable -->
```

Comment:
Why not unify the label using the page title instead of the URL?
```suggestion
* [History of Zero Trust Security | Best Introduction 2022 | Infraon](https://infraon.io/blog/history-of-zero-trust-security/) <!-- cspell:enable -->
```